### ISSUE TITLE
Support for Ansible 10.3 / 10.4

### ISSUE THREAD

**Issue by kamranayub** [spacelift-io/runner-ansible#23]
I am trying out Spacelift and my Ansible stack is working... except I'm relying on some modules that support newer parameters available in Ansible 10.3+. I noticed Alpine `edge` supports 10.4.0-rc0 but it seems like this is using the older versions, so my Run is failing.

![image](https://github.com/user-attachments/assets/6768762c-3401-4e38-bd0d-a9cefd06c4ca)

![image](https://github.com/user-attachments/assets/4efbcb38-91de-4535-8d75-6a0ba833bccf)

I was trying to figure out how to customize this image but I'm a Docker newbie. Is there a way I can either a) customize this image or b) have a tag for the latest version to use on Spacelift easily?

**Comment by kamranayub** [comment]
OK I figured it out!

**Workaround**

I can customize the image basically by copy/pasting and adjusting the versions 😄 

Here is my new runner image Dockerfile:

```
FROM public.ecr.aws/spacelift/runner-terraform:v1.5.0_20240617

USER root
WORKDIR /home/spacelift

RUN apk -U upgrade && apk add --no-cache gcc py3-pip python3-dev musl-dev libffi-dev && \
  pip install ansible==10.4.0 ansible-runner==2.4.0 --break-system-packages && \
  apk del python3-dev gcc musl-dev libffi-dev

USER spacelift
```

And it's available here:

https://github.com/users/kamranayub/packages/container/package/spacelift-ansible

I expect this is temporary until an official runner image is available with 10.4.0 🚀 

But I'm unblocked for now, it's working!

**Comment by peterdeme** [comment]
@kamranayub thanks for opening the issue. The ansible versioning should be solved in https://github.com/spacelift-io/runner-ansible/pull/21
